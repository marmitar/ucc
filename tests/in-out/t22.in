int main(int i, j)
{
  j = i+1;
  return j-i;
}
