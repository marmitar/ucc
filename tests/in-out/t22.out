ParserError: Before: j at 1:17
